frontend/src/components/MenuNav.vue

```diff
@@ -35,13 +35,7 @@
             class="navbar-item"
             v-for="(pageName, index) in getPagesByCategory(category)"
             v-bind:key="index"
-            :to="{
-              name: 'pages',
-              params: {
-                category: slugify($t(category)).toLowerCase(),
-                page: pageName.slug,
-              }
-            }"
+            :to="`/pages/${slugify($t(category)).toLowerCase()}/${pageName.slug}`"
```

Comment:
Noob question: What is the purpose of backticks inside a string?

Reply:
When there is a ":" before the attribute, it evaluates what is inside the " ". So in this case it is not a string within a string.